app/account/serializers.py

```diff
@@ -0,0 +1,30 @@
+import typing as tp
+
+from account.schemas import AccountData
+
+
+def account_serializer(model_objects: tp.Any,
```

Comment:
why Any if that's supposed to be for Account?

Reply:
circular import) in next version i removed this

Reply:
what next version?

Reply:
function should almost always be start with the verb